test/test_playbooks/domain_vars.yml

```diff
@@ -1,6 +1,5 @@
 ---
-domain_name: "example.org"
-domain_dns_proxy: "centos7-devel.example.com"
+domain_name: "test-domain.org"
```

Comment:
I'd prefer not to use a non-example domain here.

Reply:
OK. I used that to make sure, it did not fail the matching.

Reply:
Right. We have a few possibilities here (see https://tools.ietf.org/html/rfc6761), but we shouldn't use a domain that could be owned by somebody else.

Reply:
Did i? Anyway, changing it back.